group_vars/vaxtron_cluster/vars.yml

```diff
@@ -793,6 +801,7 @@ lfs_mounts:
       - name: umcg-radicon
       - name: umcg-rehabilitation
       - name: umcg-reki
+      - name: umcg-respire
```

Comment:
Group is added to this list for prm03, but did not get quota assigned in quota.yml for `prm03`: this means users will see that folder, but cannot use it; is that expected?